src/views/PackageDetail.vue

```diff
@@ -13,7 +13,12 @@
       </div>
       <div class="hero-body">
         <div class="container">
-          <h1 class="title has-text-centered">{{ $route.params.id }}</h1>
+          <h1 class="title has-text-centered pdt-module-head">
+            <a href="/gallery" class="back-arrow" :title="'Back to Gallery'">
```

Comment:
Please use the `router-link` component, for seamless navigation:
```suggestion
            <router-link to="/gallery" class="back-arrow">
```